packages/url/js/dune

```diff
@@ -3,8 +3,7 @@
  (modes melange)
  (public_name server-reason-react.url_js)
  (libraries melange.js)
- (wrapped false)
  (preprocess
-  (pps melange.ppx)))
+  (pps melange.ppx))
+ (implements url))
 
-(copy_files# "../URL.rei")
```

Comment:
We also don't like `copy_files`, but until we have the dune single-context feature, we need it.
(Related https://github.com/ml-in-barcelona/server-reason-react/pull/317/files#r2453920368)